content/documentation/device-examples/components/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Components
+weight: 50
+---
+
+The core Pi4J V.2 library doesn't contain any specific support for devices like buttons, motors, LCD etc. This page lists widely used components
+for which a Java implementation, based on Pi4J V.2, is available.
+
+### Current available components
+
+| Device(s)                     | Developed by          | Link              |
+| :---                          | :---                  | :---              |
+| [Simple Button](/documentation/device-examples/components/simplebutton) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+| [Simple LED](/documentation/device-examples/components/simpleled) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+| [AD Converter ADS1115](/documentation/device-examples/components/ads1115) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+| [Buzzer](/documentation/device-examples/components/buzzer) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+| [Camera](/documentation/device-examples/components/camera) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+| [Joystick](/documentation/device-examples/components/joystick) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+| [Joystick Analog](/documentation/device-examples/components/joystickanalog) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+| [LCD Display](/documentation/device-examples/components/lcddisplay) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+| [LED Button](/documentation/device-examples/components/ledbutton) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+| [LED Matrix](/documentation/device-examples/components/ledmatrix) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+| [LED Strip](/documentation/device-examples/components/ledstrip) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+| [Potentiometer](/documentation/device-examples/components/potentiometer) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+| [Servo](/documentation/device-examples/components/servo) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+
+### Simple Implementation
+
+For a Simple Implementation, the recommendation is to use the [Launcher Class](https://github.com/Pi4J/pi4j-example-components/blob/Dev-Arcade/src/main/java/com/pi4j/example/Launcher.java).
+With this, a simple application can be started.
+
+TODO: better statement how the launcher works
```

Comment:
Please remove TODO or even better add some extra info about the Launcher (with code example?)

Reply:
TODO is removed. How it is implemented, is shown in the example-apps